### ISSUE TITLE
Mage t7, t7.5 4 set bonus doesnt work, server Angrathar (Horde).

### ISSUE THREAD

**Issue by AdminAdministrator** [SunwellTracker/issues#1032]
Decription: The 4 set bonus for mages t7, t7,5 , "Your offensive spells gain an additional 5% increased critical strike bonus damage" does not work properly. 
I did some tests and i have facts to state that this set is bugged. 

How it works: 
I see no increase in Critical damage as fire in pve. Look tests below.

How it should work: 
"Your offensive spells gain an additional 5% increased critical strike bonus damage"

Source (you should point out proofs of your report, please give us some source): I did some reaserch on the critted frostfire bolts (max skill), and the results are as following. 10 critical strikes with the 4 set bonus and 10 without the 4 set bonus, the gear used in both tests are almost identical, the trinkets are unequiped and no buffs. Spellpower is +5 more in 4 set gear, neverless it has no major increase in dmg on critical strikes. 
Critical strike with Frostfire bolt without 4set: 6983, 6937, 7025, 7019, 6937, 7153, 7111, 7020, 6997, 6968. 
The average value is as following: 
6983+6937+7025+7019+6937+7153+7111+7020+6997+6968=70 150      

70 150/10= 7015     the average critical striket with frostfire bolt is 7015, without 4 set bonus. 

Critical strike with Frostfire bolt with 4set: 7121, 7106, 7205, 6963, 7065, 7064, 7029, 7227, 7064, 6982.
7121+7106+7205+6963+7065+7064+7029+7227+7064+6982=70,826

70 826/10=7082,6   the average critical strike with frostfire bolt is 7082,6, with 4 set bonus(and +5 more spellpower duo to gear change)

Conclusion, i only see a minor uppgrade in critical dmg. The increase is to small to be called an 5% increase, it is lower than 1% increase in dmg as for critical strike damage.  I would appreciate an investigation of this matter, if it really is bugged, then its devestating for fire mages.

**Comment by Scuffedmac** [comment]
Can confirm, I did the same tests with no avail. 

Please, this is groundbreaking for our mages, have a look at it ASAP.

**Comment by UFJ** [comment]
Still nothing done?

**Comment by iamaio** [comment]
Was this even acknowledged yet? I mean... a lot of naxx and eoe was buffed in terms of hp and enrage timer it makes sense to prioritize a bug that would increase total raid dps even if by a little percentage.

**Comment by gabbat** [comment]
I tested this with arcane missiles (as its fixed dmg, no variation) and it was working flawelessly. 
Can you try retesting with arcane missiles, fireball, frostbolt? and maybe increase your sample size, as 10 casts is literally nothing :(

**Comment by Dootz** [comment]
I also carried out some tests, and it was visibly working with frostbolt however, with fireball it wasn't that obvious (but I did only 10 crits per spell).

**Comment by UFJ** [comment]
Ok, so I have done some testing and the 4p is not bugged, it is just incredibly crappy. The thing is the 4p is an additional of 5% INCREASED critical strike bonus damage. This doesn't mean an increase of 5% damage on your crits.

Since a basic spell is 100% damage, and a base crit is 150%, it means the initial increase is 50, so the 4P increase this by 5%. Which leads to a crit doing 152.5% of the base damage when you crit with 4P. This is a meager 1.3% increased damage on your crits.

You also have to factor in that mages (especially the arcane spec) already have this kind of modifier. Both fire and arcane have a base extra 50% increased critical bonus damage. So we end up with 50 x (1+0.5+0.05) as the bonus critical damage which is 77.5, meaning a crit damage does 177.5% of the base damage when you have the 4P instead of 175% in arcane or fire (not counting ignite or the Arcane Missile glyph).

So we end up with our crits doing 2.5 points of percent more damage, this means the 4P can be resumed to an increase of 2.5 * crit%. Most mages will have around 40% crit chance which would correspond to a total increase in damage of 1%, which is laughable. A quick guess is that if you have two pieces that have 13 ilvl more than your set pieces, it probably would lead to more damage to equip those pieces rather than to keep the 4P active. Anyways, the BiS for mages in arcane is not with 4P in any case.

**Comment by Semper-Numquam** [comment]
I'm testing with 1503 SP in Frost spec, so I don't have any crit bonus modifiers in arcane missiles. No meta, no talents, no set bonus.
1503 SP -> Arcane Missile hits 790 non crit and 1185 crit every time

Now I'm testing with 1498 SP. I have no crit bonus modifiers on AM, no meta, no talents BUT I have set bonus.
1498 SP ->Arcane Missile hits 788 and 1201 crit. 

So we can appreciate that this is indeed a 2.5% increase on the damage from the crits. Since 788 x 1.5 = 1182 and 788 x 1.525 = 1201.7, we can agree that the set bonus does increase the damage by 2.5% on critical strickes.

The crit gain from AM was 394, which increased by 5% becomes 413. 788+413 = 1201 so the math is correct.

I'm testing with 1503 SP in Fire spec. I have Burnout, no meta and no set bonus. My crits are 1.75x dmg.
1503 SP -> Arcane Missile hits 814 non crit and 1424 crit every time. 814 x 1.75 = 1424.5 so we ok so far.

I'm testing with 1500 SP in Fire spec. I have Burnout, no meta BUT I have set bonus.
1500 SP ->Arcane Missile hits 813 and 1442 crit. 

813x1.75=1422.
1422-813=609 (This means the bonus damage (+75%) from the crit is 609)
1219-813=406 (This means the bonus damage without Burnout would be 406 from 1.5x dmg)
406x1.5=609 for confirmation.

So we have a bonus 609 damage from AM crits. A 5% increase on this 609 damage would mean (609÷100)x5=30.45. So Arcane Missiles would be 813 + 609 + 30 = 1452.

There's a slight difference between the 1442 crit I get on target dummy and the 1452 my lame maths tell me I should get, so my guess is that the bonus is not staking properly with other crit bonus effects? Or I failed my calculations somwhere.

Hope this data sheds some light to anyone with a smarter brain than mine, as I suck at math :cry:
